### PR TITLE
2523: Update documentation for e2e tests

### PR DESCRIPTION
**Pull request by LeandraH**
### Short description

This updates the documentation for running the e2e tests locally.

I only added how to add env variables in zshell because it's the default shell on Mac but maybe someone with a Windows machine can add their version.

### Resolved issues

Fixes: #2523 

---

### CODE REVIEW

File: docs/e2e-tests.md [hunk 1]
```diff
@@ -70,19 +70,7 @@ yarn workspace e2e test:web --spec regexp
...
-yarn workspace e2e prepare:native:start
-```
-
-It is recommended to start the app once manually to avoid timeouts during local testing. After you have the app
-installed and running you can execute the native E2E-tests with
-
-```
-yarn workspace e2e test:native
-```
+Check in the documents [here](../native/docs/e2e-testing.md).
```
Comment (sarahsporck) [inline docs/e2e-tests.md:73]:
```suggestion
Check the documents [here](../native/docs/e2e-testing.md).
```

File: native/docs/e2e-testing.md [hunk 2]
```diff
@@ -1,57 +1,44 @@
...
 
 We are using Jest and Appium to run the tests.
-The `e2e-tests/config/configs.js` contains the urls to the Appium servers and the capability definitions. A set of
-capabilities is like a requirement of devices you want to test against (e.g. Android 9/iOS 11)
-. [Here](https://www.browserstack.com/app-automate/capabilities) is a good definition of capabilities and which caps you
-can set.
+The file `e2e-tests/native/capabilities.ts` contains the capability definitions. A set of capabilities is like a
+requirement of devices you want to test against (e.g. Android 9/iOS 11).
+[Here](https://www.browserstack.com/app-automate/capabilities) is a good definition of capabilities and which
+caps you can set.
```
Comment (sarahsporck) [inline native/docs/e2e-testing.md:7]:
```suggestion
proporties you can set.
```

**Review by sarahsporck — APPROVED** [review]
Thanks a lot for fighting through this setup :) It seems that there were quite a few aspects which were not up to date anymore 😅

**Review by steffenkleinle — APPROVED** [review]
thanks :writing_hand:

**Pull request merged**